app/assets/stylesheets/custom.css.scss

```diff
@@ -75,3 +75,18 @@ h4 {
   margin-left: auto;
   margin-right: auto;
 }
+
+.item-update {
+  background: rgba(43,58,223,0.2);
+  width: 500px;
```

Comment:
Properties should be ordered background, border, border-color, border-style, margin-left, margin-right, width